stan/math/prim/fun/norm1.hpp

```diff
@@ -0,0 +1,30 @@
+#ifndef STAN_MATH_PRIM_FUN_NORM1_HPP
+#define STAN_MATH_PRIM_FUN_NORM1_HPP
+
+#include <stan/math/prim/meta.hpp>
+#include <stan/math/prim/err.hpp>
+#include <stan/math/prim/fun/Eigen.hpp>
+#include <cstddef>
+#include <vector>
```

Comment:
I don't think these headers are needed here, right?

Reply:
Good catch. Changed in `norm2` as well